src/System.Management.Automation/utils/EncodingUtils.cs

```diff
@@ -102,4 +102,22 @@ public override object Transform(EngineIntrinsics engineIntrinsics, object input
 
     }
 
+    /// <summary>
+    /// Allows to complete for the possible values of the -Encoding parameter.
```

Comment:
Suggest instead...

Provides the set of Encoding values for tab completion of an Encoding parameter.

Reply:
@dantraMSFT Thank you very much for this helpful suggestions. Fixed it.